src/runners/http_rfqm_service_runner.ts

```diff
@@ -95,8 +103,12 @@ export async function buildRfqmServiceAsync(connection: Connection, asWorker: bo
             META_TX_WORKER_MNEMONIC,
             RFQM_WORKER_INDEX,
         );
+
+        // TODO (rhinodavid): Remove once migration to ethers.js is complete
         const privateWalletSubprovider = new PrivateKeyWalletSubprovider(workerPrivateKey);
         provider = RfqBlockchainUtils.createPrivateKeyProvider(rpcProvider, privateWalletSubprovider);
+
+        ethersWallet = Wallet.fromMnemonic(META_TX_WORKER_MNEMONIC, `m/44'/60'/0'/0/${RFQM_WORKER_INDEX!}`);
```

Comment:
there's a few functions in `rfq_blockchain_utils` such as `_getPathByIndex` that could reduce duplication of the `m/44'/60'/0'/0/` stuff. That suggests that we could create a new function called `createWallet` on `rfq_blockchain_utils`

Reply:
Ack. My preference is actually to rip out all those functions once we get rid of web3-wrapper. This line is concise and clear (I think) and we can get rid of a lot of code we have to maintain.

Reply:
if you really want to.. it's not reused so my preference is to keep it inline

Reply:
I prefer to move it into a constant, but will leave it up to you. (i.e. definitely not a blocker)